share/completions/jhipster.fish

```diff
@@ -0,0 +1,37 @@
+# JHipster 4.9.0
+
+# Check if command already given
+function __fish_prog_needs_command
+    set cmd (commandline -opc)
+    echo $cmd
+    if [ (count $cmd) -eq 1 -a $cmd[1] = 'jhipster' ]
+        return 0
+    end
+    return 1
+end
+
+# Options
+complete -f -c jhipster -n '__fish_prog_needs_command' -s d -d 'Enable debugger'
+complete -f -c jhipster -n '__fish_prog_needs_command' -s h -d 'Output usage information'
+complete -f -c jhipster -n '__fish_prog_needs_command' -s V -d 'Output version number'
+
+# Commands
+complete -f -c jhipster -n '__fish_prog_needs_command' -a app -d 'Create a new JHipster application based on the selected options.'
+complete -f -c jhipster -n '__fish_prog_needs_command' -a aws -d 'Deploy the current application to Amazon Web Services.'
+complete -f -c jhipster -n '__fish_prog_needs_command' -a ci-cd -d 'Create pipeline scripts for popular Continuous Integration/Continuous Deployment tools.'
+complete -f -c jhipster -n '__fish_prog_needs_command' -a client -d 'Create a new JHipster client-side application based on the selected options..'
+complete -f -c jhipster -n '__fish_prog_needs_command' -a cloudfoundry -d 'Generate a `deploy/cloudfoundry` folder with a specific manifest.yml to deploy to Cloud Foundry.'
```

Comment:
Is it necessary to mention manifest.yml here?

I.e. would it be obvious to anyone who knows this tool that this reads manifest.yml?